app/build.gradle

```diff
@@ -66,6 +66,7 @@ android {
         buildConfigField "String", "TREETRACKER_CLIENT_SECRET", "\"${treetracker_client_secret}\""
         buildConfigField 'Boolean', 'TREE_HEIGHT_FEATURE_ENABLED', "false"
         buildConfigField 'Boolean', 'TREE_NOTE_FEATURE_ENABLED', "true"
+        buildConfigField 'Boolean', 'TREE_DBH_FEATURE_ENABLED', "false"
```

Comment:
What is a DBH value?

Reply:
https://en.wikipedia.org/wiki/Diameter_at_breast_height